__tests__/app.test.js

```diff
@@ -52,11 +51,12 @@ describe('GET', () => {
     });
   });
   describe('/api/articles/:article_id', () => {
-    test('status: 200 - should respond with an article object which should have the following properties: author, title, article_id, body, topic, created_at and votes.', () => {
+    test.only('status: 200 - should respond with an article object which should have the following properties: author, title, article_id, body, topic, created_at and votes.', () => {
       return request(app)
         .get('/api/articles/1')
         .expect(200)
         .then((response) => {
+          console.log(response.body.article)
           expect(response.body.article[0]).toEqual(
```

Comment:
I would say here that if we are only requesting a single article - the `article` property on the response body should be that article object rather than an array containing it